vulnerabilities/tests/test_nvd.py

```diff
@@ -154,6 +164,10 @@ def test_to_advisories(self):
         found_advisories = list(self.data_src.to_advisories(self.nvd_data))
         # Only 1 advisory because other advisory is hardware related
         assert len(found_advisories) == 1
-        found_advisories[0].vuln_references = sorted(found_advisories[0].vuln_references, key=lambda x: x.url)  # nopep8
+        found_advisories[0].vuln_references = sorted(
+            found_advisories[0].vuln_references, key=lambda x: x.url
+        )  # nopep8
+        # for id,ref in found_advisories[0].vuln_references :
```

Comment:
This commented code may be best deleted?

Reply:
Deleted